modules/install-osp-deploy-dualstack.adoc

```diff
@@ -24,15 +24,74 @@ DHCP must be enabled on the subnets.
 . Add the IPv6 subnet to the router to enable router advertisements. If you are using a provider network, you can enable router advertisements by adding the network as an external gateway, which also enables external connectivity.
 
 
-. To configure IPv4 and IPv6 address endpoints for cluster nodes, edit the `install-config.yaml` file. The following is an example of an `install-config.yaml` file.
+. To configure IPv4 and IPv6 address endpoints for cluster nodes, edit the `install-config.yaml` file.
++
+The following is an example of an `install-config.yaml` file for an IPv4 primary dual-stack install-config.
++
+.Example `install-config.yaml`
```

Comment:
This corresponds to an **IPv6** primary dual-stack install-config, not IPv4. Can you swap the install-configs so you first show the IPv4 primary then the IPv6 primary?